app/views/dashboards/index.html.erb

```diff
@@ -79,6 +79,8 @@
                 <% end %>
               </td>
               <td>
+                <% if result.is_a?(WasteCarriersEngine::TransientRegistration) %>
+
                 <% if result_type(result).present? %>
```

Comment:
Can we fix indentation? :D

Reply:
I think this gets removed by another PR anyway as it was a temporary fix :)